CryptoPkg/Library/BaseCryptLibMbedTls/Pk/CryptPkcs7VerifyCommon.c

```diff
@@ -555,7 +557,11 @@ MbedtlsPkcs7SignedDataVerifySigners (
   ZeroMem(Hash, MBEDTLS_MD_MAX_SIZE);
   mbedtls_md (MdInfo, Data, DataLen, Hash);
   if (SignerInfo->AuthAttr.p != NULL) {
+    TempAuthAttr = *(SignerInfo->AuthAttr.p);
+    *(SignerInfo->AuthAttr.p) = 0x31;
     mbedtls_md (MdInfo, SignerInfo->AuthAttr.p, SignerInfo->AuthAttr.len, Hash);
+    *(SignerInfo->AuthAttr.p) = TempAuthAttr;
+    *(SignerInfo->AuthAttr.p) = 0xA0;

```

Comment:
why override to 0xA0?

Reply:
Thanks.  I have submitted the PR:  https://github.com/tianocore/edk2-staging/pull/459